auacm/app/modules/competition_manager/views.py

```diff
@@ -18,13 +24,26 @@ def get_competitions():
     past = list()
     upcoming = list()
     current_time = int(time())
+
+    registered_rows = session.query(CompUser).filter(
+            CompUser.username == current_user.username).all()
+    registered = set()
+    for row in registered_rows:
+        registered.add(row.cid)
+
     for competition in session.query(Competition).all():
         if competition.stop < current_time:
-            past.append(competition.to_dict())
+            past.append(competition.to_dict(
+                user_registered=competition.cid in registered
+            ))
```

Comment:
Just want to note that [Google style](https://google.github.io/styleguide/pyguide.html?showone=Indentation#Indentation) has closing parens for function calls on the same line